src/swagger/api.yaml

```diff
@@ -245,8 +245,6 @@ paths:
       produces:
         - application/json
       responses:
-        '200':
```

Comment:
This is to sync with open api spec change at https://github.com/okta/openapi/pull/227